module/Checker/view/email/graduate-renewal.phtml

```diff
@@ -0,0 +1,18 @@
+<p>Dear <?= $firstName ?>,</p>
+<p>
+    You are currently registered as a Graduate (<i>Afgestudeerde</i>) at GEWIS until <b><?= $currentExpiration->format('F jS, Y') ?></b>. 
+    We care about your privacy, so we'd like to confirm that you want to renew this graduate status.
+</p>
+<p>
+    Please click <a href="<?= $url ?>" style="color: #C40000; text-decoration: none;" target="_blank" rel="noopener nofollow">here</a> to review your personal details and renew your status as graduate of GEWIS until <b><?= $newExpiration->format('F jS, Y') ?></b>.
+    If you do not click this link, your status will automatically expire and we will delete your data in the way described in our <a href="https://gewis.nl/association/regulations/privacy-statement" style="color: #C40000; text-decoration: none;" target="_blank" rel="noopener nofollow">privacy statement</a>.
+</p>
+<p>
+    If you prefer this, you can also renew your membership by replying to this email.
+    Please keep the subject intact so we can process it quicker.
+</p>
+<p>
+    On behalf of the the board,
+    <br><br>Daan Verkade
+    <br><i>Secretary of GEWIS 2022-2023</i>
```

Comment:
This is about the `TODO` in the `Renewal` service, we can make this more of a mess by getting the current secretary.

For now this is fine, but I do not want to make sure that someone keeps this up-to-date every year (especially after we have left GEWIS).

Reply:
I did not want to make the assumption that you always had a secretary installed in the database, thus I thought of making a global config menu in a separate PR where you could set among other things this information.

Then we could also use it for for example the registration emails. (perhaps the secr. wants to promote the FLUP there again)

I also did not want to do a "kind regards, the secretary" because that was slightly too unpersonal. If you were to receive an email to renew your membership with that, it does not sound very welcoming.

Reply:
> I did not want to make the assumption that you always had a secretary installed in the database

We should implement GH-92, not that fixes the 'secretary' part of this.